genestack_client/settings/genestack_user.py

```diff
@@ -24,25 +30,16 @@ def _get_server_url(host):
     split_host = urlsplit(host)
     has_scheme = bool(split_host.scheme)
 
-    # compatibility with dev settings
-    # TODO Add code that will do migration in configs and remove this check.
-    if host.startswith('localhost'):
-        return 'http://%s/frontend/endpoint' % host
-
     # default to HTTPS if not explicitly defined
     url_stub = host if has_scheme else 'https://%s' % host
-    # trust provided path if it ends with '/frontend'
     url_stub = url_stub.rstrip('/')
+    # trust provided path if it ends with '/frontend'
```

Comment:
Why exactly we trust it?

Reply:
Wasn't invented by me, so my assumption: since host address already ends with /frontend, a client might know the ODM address and some details about ODM urls. It doesn't make any sense to convert the path further, e.g. add '/endpoint' to it, we can just use it as it was provided by client, without additional connection checks.

Reply:
Mikhail is correct, the idea was that the path containing `/frontend` is a correct one, or at least we don't have a clear way to guess the correct URL